components/fxa-client/src/internal/config.rs

```diff
@@ -204,24 +148,104 @@ impl Config {
         Url::parse(&self.remote_config()?.authorization_endpoint).map_err(Into::into)
     }
 
+    pub fn token_endpoint(&self) -> Result<Url> {
+        Url::parse(&self.remote_config()?.token_endpoint).map_err(Into::into)
+    }
+
+    pub fn introspection_endpoint(&self) -> Result<Url> {
+        Url::parse(&self.remote_config()?.introspection_endpoint).map_err(Into::into)
+    }
+
+    pub fn userinfo_endpoint(&self) -> Result<Url> {
+        Url::parse(&self.remote_config()?.userinfo_endpoint).map_err(Into::into)
+    }
+}
+
+#[allow(dead_code)]
+// Methods that aren't actually used.  Should we delete these?
+impl Config {
```

Comment:
I feel like we should delete, but I don't know about the history of these.  What do others think?

Reply:
I'm always up for killing unused code 🔥